azure-core/azure/core/__init__.py

```diff
@@ -39,7 +38,6 @@
 ]
 
 try:
-    from .pipeline_client_async import AsyncPipelineClient
```

Comment:
This import actually needs to be here. You can just add a pylint disable for this line